topics/go/algorithms/data/heap/heap.go

```diff
@@ -0,0 +1,140 @@
+// Package heap implements a heap interface, and add ability to store values of type Data in the minimum Heap.
```

Comment:
Keep comments within a 80 column with.
All comments must you proper sentence structure, punctuation and grammar.
Inside of a function, keep the comment width inside the function.

Please check the code for all these comment rules.

Reply:
Oh, just forget to set up my IDE column width. Thank you.